cfgov/v1/models/base.py

```diff
@@ -217,6 +220,29 @@ def specific_categories_query(block, parent_slug):
         return {search_type: queryset for search_type, queryset in
                 related.items() if queryset}
 
+    def flush(self):
+        if not settings.ENABLE_AKAMAI_CACHE_PURGE:
+            return False
+        auth, fast_purge_url = get_akamai_credentials()
+        page_relative_path = self.url_path.replace("cfgov/", "")
```

Comment:
Okay it works again, must've been something with my environment.  This gets called after a page if published, so we can't skip flushing new pages that way.  I can add that to my TODO list but the complex logic might not be worth it.